debian/rules

```diff
@@ -22,6 +22,10 @@ ifeq ($(DEB_HOST_ARCH), armel)
   extraopts += -DWITH_ATOMIC_OPS=OFF
 endif
 
+ifeq (,$(filter $(DEB_HOST_ARCH), s390 s390x))
```

Comment:
@smithfarm see https://www.gnu.org/software/make/manual/html_node/Text-Functions.html. if it matches, a `s390` or `s390x` is returned, which is non-empty. in that case we should pass `-DWITH_RADOSGW_BEAST_FRONTEND=OFF` to cmake.